Add webhook export option (#3535)
astronomer/docs: astro/reporting-dash-exports.md

3. Choose the file format that you want to download your data in. Astro generates your data export and saves it to your local computer.

## Export reporting data

> Instead of downloading your data directly, you can choose to email it to a specified address or configure a webhook to send it to.

Reviewer: ```suggestion
Instead of downloading your data directly, you can email it to a specified address or an external service through a webhook.
```

1. To view Astro reporting dashboards, click your Workspace name in the upper left corner of the Astro UI, click **Organization Settings**, and then click **Dashboards**. You can also access this page directly at [https://cloud.astronomer.io/dashboards](https://cloud.astronomer.io/dashboards).
2. Open the **Export** menu and then select **Send now**.

    <img src={require("../static/img/docs/dash-open-menu.png").default} alt="Expand the menu to view all options for downloading data, emailing data, scheduling a data export, and setting up a data alert." style={{ width: "60%", maxWidth: "400px", height: "auto" }} />

3. Enter the where you want to send the data report.
    - **Email** - Enter the recipient's email address and optionally edit the subject line and email body message.

>     - **Webhook** - ([Sigma Beta](https://help.sigmacomputing.com/docs/webhook-exports)) Add the webhook URL where you want to export your data to.

Reviewer: ```suggestion
    - **Webhook** - ([Sigma feature in beta](https://help.sigmacomputing.com/docs/webhook-exports)) Add the webhook URL where you want to export your data to.
```
Might just be me but Sigma Beta with no other context is giving tiktok hustlepreneur slang

When you receive a report via email, the sender appears as **Sigma Computing**. The subject line also includes the name of the dashboard element's data you exported.

![Example email alert shows the sender as Sigma with information about the report in the subject line.](/img/docs/dash-email-alert.png)

### Schedule a data report

1. To view Astro reporting dashboards, click your Workspace name in the upper left corner of the Astro UI, click **Organization Settings**, and then click **Dashboards**. You can also access this page directly at [https://cloud.astronomer.io/dashboards](https://cloud.astronomer.io/dashboards).
2. Open the **Export** menu for the reporting element you want to schedule an export for and select **Schedule exports...**.

    <img src={require("../static/img/docs/dash-open-menu.png").default} alt="Expand the menu to view all options for downloading data, emailing data, scheduling a data export, and setting up a data alert." style={{ width: "60%", maxWidth: "400px", height: "auto" }} />

3. Enter the where you want to send the data report.
    - **Email** - Enter the recipient's email address and optionally edit the subject line and email body message.

>     - **Webhook** - ([Sigma Beta](https://help.sigmacomputing.com/docs/webhook-exports)) Add the webhook URL where you want to export your data to.

Reviewer: ```suggestion
    - **Webhook** - ([Sigma feature in beta](https://help.sigmacomputing.com/docs/webhook-exports)) Add the webhook URL where you want to export your data to.
```
